### ISSUE TITLE
Confetti on Windows

### ISSUE THREAD

**Issue by BruceEckel** [aviaviavi/confetti#7]
The readme says "You can install from source with stack." Stack does work on Windows, but it would be nice to indicate whether it works so people know whether to try or not. Thanks.

**Comment by aviaviavi** [comment]
Right, thanks for pointing this out! I rather doubt this works on Windows as-is due to the file system operations confetti does, but it could probably be tweaked without _too_ much effort. I don't plan to add this support myself any time soon, but would definitely love a PR from anyone wanting to do so!

I'll get the readme updated to include this info.

**Comment by BruceEckel** [comment]
Just tried installing stack and running stack build, got:

>stack build

Error: While constructing the build plan, the following exceptions were
encountered:

In the dependencies for unix-2.7.2.1:
    unbuildable must match <0, but the stack configuration has no specified
version
needed due to confetti-0.3 -> unix-2.7.2.1

Plan construction failed.


-- Bruce Eckel
www.MindviewInc.com <http://www.mindviewinc.com/>
Blog: BruceEckel.github.io
www.WinterTechForum.com
www.AtomicScala.com
www.Reinventing-Business.com
http://www.TrustOrganizations.com <http://www.ScalaSummit.com>

**Comment by aviaviavi** [comment]
Yeah, for Windows support that package would definitely need to be swapped out for something cross platform. The `filepath` package also is being used for some posix features, though it does have a complete parity with the Windows API so that swap should in theory be straightforward.

**Comment by aviaviavi** [comment]
README is updated. I'm going to rename this issue so we can use this as a place for general Windows support discussion. Thanks again for bringing this up @BruceEckel!
